thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import peakdetection as pkd
+from IPython import embed
```

Comment:
from IPython import embed
is for debugging only. For the package it adds a unnecessary dependency on ipython. Please remove here and everywhere else.